docs/developer-manual/archived-docs.md

```diff
@@ -30,36 +30,38 @@ consultancy. The guidelines and specification in COOP are free, open-source, and
 can be implemented by any other project seeking to develop their own Cardano
 oracle feeds.
 
-Shortly after this version was released, IOG discontinued support for both
-the Plutus Application Backend (PAB) and Plutus-Chain-Index. Both were critical
+Shortly after this version was released, IOG discontinued support for both the
+Plutus Application Backend (PAB) and Plutus-Chain-Index. Both were critical
 components in the COOP v0 stack. Aware of the discontinued support, the Orcfax
 team partnered with technical consultants through MLabs to replace the
 components. Unfortunately, the plutus-chain-index component failed before a
 replacement could be rolled-out.
 
 And so v0 was suspended in May of 2024.
 
-More information pertaining to this failure can be found in both the
-[public announcement][medium-1] and in the subsequent incident
-[report][incident-35].
+More information pertaining to this failure can be found in both the [public
+announcement][medium-1] and in the subsequent incident [report][incident-35].
 
-Users and developers may find it informative to review the entire
-v0 [Incidents][incidentRepo-1] repository as it contains many critical learnings
+Users and developers may find it informative to review the entire v0
+[Incidents][incidentRepo-1] repository as it contains many critical learnings
 from the implementation of v0.
 
 [coop-1]: https://github.com/mlabs-haskell/cardano-open-oracle-protocol/
 [utxo-1]: https://docs.cardano.org/learn/eutxo-explainer/
-[cip-31]: https://docs.cardano.org/about-cardano/evolution/upgrades/vasil/#reference-inputs-cip-31
-[cip-32]: https://docs.cardano.org/about-cardano/evolution/upgrades/vasil/#inline-datums-cip-32
+[cip-31]:
+    https://docs.cardano.org/about-cardano/evolution/upgrades/vasil/#reference-inputs-cip-31
```

Comment:
its not a problem that it breaks these links like this?

Reply:
Nope, not as far as I can tell from running it locally and clicking the links.